src/views/Home.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="wrapper">
+    <!-- <Header /> -->
+    <HelloWorld msg="Router/Page based content goes here. " />
+    <div id="learning-site-videos">
+      <div class="video-grid-container">
+        <LearningVideo
+          v-for="item in videoList"
+          v-bind:video="item"
+          v-bind:key="item.id"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+// @ is an alias to /src
+import HelloWorld from '@/components/HelloWorld.vue';
+import LearningVideo from '@/components/LearningVideo/index.vue';
+
+export default {
+  name: 'Home',
+  components: {
+    HelloWorld,
+    LearningVideo,
+  },
+  data() {
+    return {
+      videoList: [
```

Comment:
Can we extract this into a different file to mock a fetch-request/response scenario?

Reply:
Ideally, there will be an api call for this.
For now, moved the videoList to `src/constants/index.js`